.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+install:
+  - pip install -r requirements.txt
+  - python setup.py install
+  - pip install pep8
+script: 
+  - ./test/tsht
+  - pep8 hocr-*
```

Comment:
`setup.py` should also be added to check. But I really would suggest to look into this afterwards. Testing currently for this coding standard will just produce a lot of errors and warnings...